### ISSUE TITLE
About the number of validation set batches

### ISSUE THREAD

**Issue by srdfjy** [wenet-e2e/wenet#990]
Hi，My training data is as follows, why is the total cv batch num 3000+, shouldn't it be 507 (65011/16/8)?

cv wav num：65011
batch_size：16
gpu card num：8

---

2022/03/24 13:46:23 2022-03-24 13:46:23,117 DEBUG CV Batch 5/3700 loss 74.450745 loss_att 79.250183 loss_ctc 63.252052 history loss 58.864828 rank 3
2022/03/24 13:46:25 2022-03-24 13:46:25,233 DEBUG CV Batch 5/3700 loss 74.450745 loss_att 79.250183 loss_ctc 63.252052 history loss 58.864828 rank 2
2022/03/24 13:46:27 2022-03-24 13:46:27,597 DEBUG CV Batch 5/3300 loss 38.618885 loss_att 41.560642 loss_ctc 31.754784 history loss 58.528153 rank 1
2022/03/24 13:46:28 2022-03-24 13:46:28,028 DEBUG CV Batch 5/3800 loss 30.252905 loss_att 31.613735 loss_ctc 27.077635 history loss 58.772122 rank 3
2022/03/24 13:46:28 2022-03-24 13:46:28,462 DEBUG CV Batch 5/3700 loss 74.450745 loss_att 79.250183 loss_ctc 63.252052 history loss 58.864828 rank 0
2022/03/24 13:46:30 2022-03-24 13:46:30,107 DEBUG CV Batch 5/3800 loss 30.252905 loss_att 31.613735 loss_ctc 27.077635 history loss 58.772122 rank 2
2022/03/24 13:46:32 2022-03-24 13:46:32,519 DEBUG CV Batch 5/3400 loss 26.543968 loss_att 28.197788 loss_ctc 22.685053 history loss 58.660158 rank 1
2022/03/24 13:46:32 2022-03-24 13:46:32,702 DEBUG CV Batch 5/3900 loss 15.008497 loss_att 16.364504 loss_ctc 11.844481 history loss 58.472115 rank 3
2022/03/24 13:46:33 2022-03-24 13:46:33,529 DEBUG CV Batch 5/3800 loss 30.252905 loss_att 31.613735 loss_ctc 27.077635 history loss 58.772122 rank 0
2022/03/24 13:46:34 2022-03-24 13:46:34,735 DEBUG CV Batch 5/3900 loss 15.008497 loss_att 16.364504 loss_ctc 11.844481 history loss 58.472115 rank 2
2022/03/24 13:46:37 2022-03-24 13:46:37,417 DEBUG CV Batch 5/4000 loss 7.200318 loss_att 7.292255 loss_ctc 6.985798 history loss 58.421397 rank 3
2022/03/24 13:46:37 2022-03-24 13:46:37,850 DEBUG CV Batch 5/3500 loss 6.339920 loss_att 6.542535 loss_ctc 5.867151 history loss 59.075062 rank 1
2022/03/24 13:46:38 2022-03-24 13:46:38,497 INFO Epoch 5 CV info cv_loss 58.350460848068494
2022/03/24 13:46:38 2022-03-24 13:46:38,486 DEBUG CV Batch 5/3900 loss 15.008497 loss_att 16.364504 loss_ctc 11.844481 history loss 58.472115 rank 0
2022/03/24 13:46:38 2022-03-24 13:46:38,500 INFO using accumulate grad, new batch size is 12 times larger than before
2022/03/24 13:46:38 2022-03-24 13:46:38,498 INFO Epoch 6 TRAIN info lr 0.0001452
